src/test_util/test_util.h

```diff
@@ -92,6 +93,10 @@ class stop_watch
 
 void create_local_test_file(const std::string &full_name, dsn::replication::file_meta *fm);
 
+// Generate a file under `file_path` with size `file_size`. The file is filled with
+// `file_size` 'a's.
+void generate_test_file(const std::string &file_path, int64_t file_size);
```

Comment:
Will the file to be deleted after the tests?

Reply:
OK, I'll implement a class managing the file's life cycle with RAII.